diff --git a/libcpp/expr.c b/libcpp/expr.c
index 2ae9be0..30784bd 100644
--- a/libcpp/expr.c
+++ b/libcpp/expr.c
@@ -797,12 +797,12 @@ cpp_classify_number (cpp_reader *pfile, const cpp_token *token,
 				? N_("use of C++11 long long integer constant")
 		                : N_("use of C99 long long integer constant");
 
-	  if (CPP_OPTION (pfile, c99))
+/*	  if (CPP_OPTION (pfile, c99))
             cpp_warning_with_line (pfile, CPP_W_LONG_LONG, virtual_location,
 				   0, message);
           else
             cpp_pedwarning_with_line (pfile, CPP_W_LONG_LONG,
-				      virtual_location, 0, message);
+				      virtual_location, 0, message);*/
         }
 
       result |= CPP_N_INTEGER;
