diff --git a/build-devkit.sh b/build-devkit.sh
index af47873..b8518fc 100755
--- a/build-devkit.sh
+++ b/build-devkit.sh
@@ -230,8 +230,8 @@ if [ ! -z $CROSSBUILD ] && grep -q "mingw" <<<"$CROSSBUILD" ; then
 	cp -v	$CROSSBINPATH/libwinpthread-1.dll $prefix/libexec/gcc/$target/$GCC_VER/
 fi
 
-echo "stripping installed binaries"
-. ./strip_bins.sh
+# echo "stripping installed binaries"
+# . ./strip_bins.sh
 
 #---------------------------------------------------------------------------------
 # Clean up temporary files and source directories
diff --git a/dkarm-eabi/scripts/build-crtls.sh b/dkarm-eabi/scripts/build-crtls.sh
index 3b13e4e..2ad5b22 100755
--- a/dkarm-eabi/scripts/build-crtls.sh
+++ b/dkarm-eabi/scripts/build-crtls.sh
@@ -14,6 +14,7 @@ cd $BUILDDIR
 mkdir -p rules
 cd rules
 tar -xvf $SRCDIR/devkitarm-rules-$DKARM_RULES_VER.tar.xz
+patch Makefile $SRCDIR/rules_arm_makefile_install.diff
 make install
 
 #---------------------------------------------------------------------------------
@@ -24,5 +25,5 @@ cd $BUILDDIR
 mkdir -p crtls
 cd crtls
 tar -xvf $SRCDIR/devkitarm-crtls-$DKARM_CRTLS_VER.tar.xz
+patch Makefile $SRCDIR/crlts_arm_makefile_install.diff
 make install
-
